packages/mdc-checkbox/foundation.js

```diff
@@ -220,6 +257,10 @@ export default class MDCCheckboxFoundation extends MDCFoundation {
     }
   }
 
+  /**
+   * @return {!HTMLInputElement|InputPropsType}
```

Comment:
Removed `HTMLInputElement` and moved `InputPropsType` def to the adapter so that the result of `querySelector()` can be casted to this type in the component class.

Returning that fake element seems to be required by some unit tests verifying the getters still work when no native control is returned.